18_regexp_lab/03.py

```diff
@@ -13,3 +13,16 @@
     Hampton,Witt,witthampton@zaphire.com
     Grant,Morgan,morgangrant@lotron.com
 """
+
+import sys
+import re
+
+(_,filename) = sys.argv
+
+with open(filename, "r") as f:
+    for line in f:
+        fields = re.search(r'^(.*),(.*),(.*)$',line)

```

Comment:
Nice! Though I think you can simplify the code by using `sub` instead of `search`.